frameworks/sveltekit/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kit-prisma",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build && cp node_modules/prisma/*query* .vercel_build_output/functions/node/render/;cp prisma/schema.prisma .vercel_build_output/functions/node/render/",
```

Comment:
Uh, just noticed this. What does this do exactly @tnzk? Is that needed to make sveltekit projects work on Vercel?

Reply:
adapter-vercel doesn't copy entire node_modules but bundles them to deploy on Lambda, so we need explicitly copy the query engine binary to make it work. I also not entirely sure and don't think super intuitive though.

Reply:
I think we should remove all this from the `package.json` then here and maybe you open an issue about it in prisma/prisma? I think we want to know about this and figure out if there is a better way. (Ok, we _definitely_ want to know and improve this - who am I kidding 😛 )

Reply:
Of course, I will open it in next week so that it can be with a quick repro. It's been quite some time since I use this workaround blindly, so might be fixed in latest Prisma.